custom_components/frigate/config_flow.py

```diff
@@ -60,7 +60,10 @@ async def async_step_user(
 
         try:
             session = async_create_clientsession(self.hass)
-            client = FrigateApiClient(user_input[CONF_URL], session)
+            info = await system_info.async_get_system_info(self.hass)
+            client = FrigateApiClient(
+                user_input[CONF_URL], info.get("timezone", "utc"), session
```

Comment:
(This is an example where you wouldn't need to pass in the timezone if it was part of the recording call)